docs/create-handlers.md

```diff
@@ -2,15 +2,330 @@
+
+#### Higher time could help low spec PCs:
+Line | Description
+---- | ----------
+Game.PauseBetweenStarts = 20; | Pause between game intances starts in milliseconds.
+
+### Focus
+
+All player's game instances must think they are the currently active window to receive input and play.
+
+:::note
+These must be always present. ForceFocus is not always needed on true. Try to see if there are any differences if you set it on false.
+:::
+
+Line | Description
+---- | ----------
+Game.Hook.ForceFocus = true;    |If our custom x360ce xinput DLL should hook into the game's window and fake Window's events so we never leave focus.
+Game.Hook.ForceFocusWindowName = "Game window title"; | If force focus is enabled, this is the window we are attaching ourselves to and the window we are going to keep on top. Needs the exact name of the window. If force focus is disabled you still need to set the game window name for resizing and positioning.
+
+If the game window title changes at some point or you want to use only a portion of the title text.
+Line | Description
+---- | ----------
+Game.HasDynamicWindowTitle = true;  | This will use only the text specified in ForceFocusWindowName. Work-around for ForceFocusWindowName having to match 1:1 with game window title for resizing, positioning, and focus: default: false
+
+If you hear audio only from one instance try adding this line.
+Line | Description
+---- | ----------
+Game.HookFocus = true; | Enable or disable hooks to trick the game into thinking it has focus \| default: false
+
+If only one instance has focus and the other ones has no input try to add this one.
+Line | Description
+---- | ----------
+Game.FakeFocus = true; | Enable or disable the sending of focus messages to each game window at a regular interval \| default: false
+
+or use this one INSTEAD (this will block keyboard/mouse input).
+Line | Description
+---- | ----------
+Game.PreventWindowDeactivation = true; | Blocks the processing of the windows message that gets sent when the window loses focus
+
+If only one instance has high FPS try to add this one (*for gamepads).
+Line | Description
+---- | ----------
+Game.SetForegroundWindowElsewhere = true; | Set the foreground window to be something other than game windows
+
+This line could help if only the first instance starts.
+Line | Description
+---- | ----------
+Game.HookInit = true; | Enable or disable hooks of functions some games may try and use to prevent multiple instances from running \| default: false
+
+### Input
+
+Single keyboard (you can set it on false if you use multiple keyboards/mice lines):
+Line | Description
+---- | ----------
+Game.SupportsKeyboard = true; | If enabled, will allow the keyboard to be used as a device input in Nucleus. Before enabling it by default, be sure that the keyboard/mouse works without problems.
+Game.KeyboardPlayerFirst = false; | Keyboard player will be the first to be initialized
+
+Gamepads:
```

Comment:
change to: 
`#### Gamepads`

This makes it easier to reference/link to, for support questions